idea/idea-android/src/org/jetbrains/kotlin/android/configure/AndroidGradleModelFacade.kt

```diff
@@ -61,7 +61,7 @@ class AndroidGradleModelFacade : KotlinGradleModelFacade {
     override fun getDependencyModules(ideModule: DataNode<ModuleData>, gradleIdeaProject: IdeaProject): Collection<DataNode<ModuleData>> {
         val ideProject = ideModule.parent as DataNode<ProjectData>
         ExternalSystemApiUtil.find(ideModule, AndroidProjectKeys.JAVA_MODULE_MODEL)?.let { javaModuleModel ->
-            val moduleNames = javaModuleModel.data.javaModuleDependencies.map { it.moduleName }
+            val moduleNames = javaModuleModel.data.javaModuleDependencies.map { it.moduleName }.toHashSet()
```

Comment:
FYI Instead of `map().toHashSet()` it's more efficient to use `.mapTo(HashSet())`

Reply:
thanks!